CONTRIBUTING.md

```diff
@@ -42,6 +42,12 @@ cargo make tests
 
 `cargo make tests` will automatically download Geckodriver to a temporary location if it isn't in the PATH.
 
+Because Geckodriver looks for `firefox` in the path, if you use
+FireFox Developer Edition, you may get an error, because Developer Editions
+binary is called `firefox-developer-edition` on some Linux distributions.
+To fix this, either install the standard version of Firefox or symlink
+`/usr/bin/firefox` to `/usr/bin/firefox-developer-edition`.
```

Comment:
I think this is too specific. This isn't true for all Linux distributions and especially not if we count other UNIX-like OSs like FreeBSD.
If the user understands what a symlink is I'm sure we can expect them to figure out the appropriate path for their OS too.
```suggestion
`firefox` to `firefox-developer-edition`.
```

Reply:
Thats fair :+1: